examples/landmarks/form.html

```diff
@@ -143,26 +143,26 @@ <h3>Source Code</h3>
                   <div class="code">
```

Comment:
Should this be an actual code block instead of escaped HTML in a div?

Reply:
Not in this PR. 🙂 Feel free to file an issue, though I guess the landmark pages need an overhaul and this would be fixed as part of that when we get to it.